open_feature/open_feature_api.py

```diff
@@ -1,10 +1,12 @@
 import typing
 
+from open_feature.evaluation_context.evaluation_context import EvaluationContext
 from open_feature.exception.exceptions import GeneralError
 from open_feature.open_feature_client import OpenFeatureClient
 from open_feature.provider.provider import AbstractProvider
 
 _provider = None
+_evaluation_context = EvaluationContext()
```

Comment:
Should this live in the `open_feature_evaluation_context` module?

Reply:
Ah, I think it does? This can probably be removed?

Reply:
Will remove it, I must have missed it when I moved it to it's own module